spec/classes/puppet_agent_osfamily_aix_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 
-describe 'puppet_agent', :unless => Puppet.version < "3.8.0" || Puppet.version >= "4.0.0" do
+describe 'puppet_agent', :unless => Puppet.version < "3.8.0" do
```

Comment:
We can stop guarding against version less than 3.8. Missed that in the commit that dropped the 3.8 guard.